bin/i18n/resources/dashboard/blocks/sync_in.rb

```diff
@@ -57,7 +57,7 @@ def prepare
           end
 
           def redact
-            I18nScriptUtils.copy_file(I18N_SOURCE_FILE_PATH, I18N_BACKUP_FILE_PATH)
+            RedactRestoreUtils.backup_source_file(I18N_SOURCE_FILE_PATH)
```

Comment:
Here is an example of where it was using `I18N_BACKUP_FILE_PATH` and it calls the function which effectively assumes `I18N_BACKUP_DIR_PATH` is equal to `I18N_ORIGINAL_DIR_PATH`. I wonder if there are any implications?

Reply:
It sounds like it would be safer to establish the `I18N_BACKUP_FILE_PATH` for each resource in the module itself instead of assuming it will exist.